tests/SessionTest.php

```diff
@@ -83,34 +101,10 @@ public function testRestart()
         $this->session->restart();
     }
 
-    public function testVisitWithoutRunningSession()
```

Comment:
I removed the test as I added the tests for the start() method.
As start() is tested for both options I thought it is enough to test just that start() 
is called in e.g. the visit method.